hermes-console/src/dummy/topic-form.ts

```diff
@@ -43,7 +43,8 @@ export const dummyTopicFormValidator = {
   ownerSource: [required()],
   owner: [required()],
   contentType: [required()],
-  retentionTimeDuration: [required(), min(0), max(7)],
+  retentionTimeDurationDays: [required(), min(0), max(7)],
+  retentionTimeDurationHours: [required(), min(0), max(7)],
```

Comment:
in `hermes-console/src/composables/topic/use-form-topic/useFormTopic.ts` we used max of 24, is this intentional?

Reply:
typo 😄 fixed